testsuites/basic/lsvmbus.py

```diff
@@ -0,0 +1,82 @@
+import math
+
+import asserts
+
+from lisa import TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.environment import EnvironmentStatus
+from lisa.features import SerialConsole
+from lisa.testsuite import simple_requirement
+from lisa.tools import Lscpu, Lsvmbus
+
+
+class VmbusNames:
+    def __init__(self, is_gen1: bool) -> None:
+        self.vmbus_names = [
```

Comment:
since the class name is `VmbusNames` already, the variable can be called `names`.